README.md

```diff
@@ -330,6 +330,8 @@ function Page2() {
 When `Page1` mounts, event with data `{page: 'Page1', event: 'pageview'}` will be dispatched.
 When `Page2` mounts, nothing will be dispatched.
 
+_**Note:** The `options.process` function does not currently take single-page app (SPA) navigation into account. If the example above were implemented as an SPA, navigating back to `Page1`, with no page reload, would **not** cause `options.process` to fire a second time even if the `Page1` component remounts._
```

Comment:
Could we also mention the workaround and maybe link to the open issue for more reference/info?

Reply:
Yup! Will do.

Reply:
Added. Wasn't sure whether you wanted to link directly to the code sandbox.